Orm/Xtensive.Orm.Tests/Storage/AsyncQueries/ExecuteAsyncExtensionTest.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Copyright (C) 2019 Xtensive LLC.
+// All rights reserved.
+// For conditions of distribution and use, see license.
+// Created by: Alexey Kulakov
+// Created:    2019.09.12
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Xtensive.Core;
+using Xtensive.Orm.Configuration;
+using Xtensive.Orm.Tests.Storage.ConcurrentCommandProcessorExecutionModel;
+
+namespace Xtensive.Orm.Tests.Storage
+{
+  public class ExecuteAsyncExtensionTest : AutoBuildTest
+  {
+    protected override DomainConfiguration BuildConfiguration()
+    {
+      var configuration = base.BuildConfiguration();
+      configuration.Types.Register(typeof(TestEntity).Assembly, typeof(TestEntity).Namespace);
+      configuration.UpgradeMode = DomainUpgradeMode.Recreate;
+      return configuration;
+    }
+
+    protected override void PopulateData()
+    {
+      using (var session = Domain.OpenSession())
+      using (var transaction = session.OpenTransaction()) {
+        var container = new EntitySetContainer(session);
+        foreach (var i in Enumerable.Range(-100, 200)) {
+          container.EntitySet.Add(new TestEntity(session) {Value = i});
+        }
+        transaction.Complete();
+      }
+    }
+
+    // [Test]
+    // public async Task PureEnumerableTest()
```

Comment:
New commented tests in this file. Delete or uncomment?

Reply:
Fixed.